src/explorer/LeetCodeTreeDataProvider.ts

```diff
@@ -42,7 +42,9 @@ export class LeetCodeTreeDataProvider implements vscode.TreeDataProvider<LeetCod
             label: element.isProblem ? `[${element.id}] ${element.name}` : element.name,
             tooltip: this.getSubCategoryTooltip(element),
             collapsibleState: element.isProblem ? vscode.TreeItemCollapsibleState.None : vscode.TreeItemCollapsibleState.Collapsed,
-            contextValue: element.isProblem ? "problem" : element.id.toLowerCase(),
+            contextValue: element.isProblem
+                ? element.isFavorite ? "problem-favorite" : "problem"
```

Comment:
Consider split the logic of handling category and problem node

Reply:
Reference:
https://github.com/Vigilans/vscode-leetcode/blob/7480bc6720550a03969d78d4a828182a727cb343/src/explorer/LeetCodeTreeDataProvider.ts#L45